blockchain/state/statedb.go

```diff
@@ -109,16 +120,25 @@ func New(root common.Hash, db Database) (*StateDB, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &StateDB{
+	sdb := &StateDB{
 		db:                       db,
 		trie:                     tr,
+		snaps:                    nil,
 		stateObjects:             make(map[common.Address]*stateObject),
 		stateObjectsDirtyStorage: make(map[common.Address]struct{}),
 		stateObjectsDirty:        make(map[common.Address]struct{}),
 		logs:                     make(map[common.Hash][]*types.Log),
 		preimages:                make(map[common.Hash][]byte),
 		journal:                  newJournal(),
-	}, nil
+	}
+	if sdb.snaps != nil {
```

Comment:
This will not be executed because snaps is always nil. Is there any more implementation done for this? Then, please add a TODO comment.

Reply:
Right. `snaps` will be inserted with a parameter. I will add a comment.